ocpmodels/datasets/ase_datasets.py

```diff
@@ -0,0 +1,355 @@
+from pathlib import Path
+import ase
+import warnings
+import numpy as np
+from torch import tensor
+
+from torch.utils.data import Dataset
+from tqdm import tqdm
+
+from ocpmodels.common.registry import registry
+from ocpmodels.preprocessing import AtomsToGraphs
+
+
+def apply_one_tags(atoms, skip_if_nonzero=True, skip_always=False):
```

Comment:
Yes, the reason for making this the default is so that GemNet-OC will work with untagged structures. I added some explanation about this to the docstring.